spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -777,9 +777,13 @@ def main(argv: Sequence[str]):
         printv('\nRemove unused label on template. Keep only label present in the input label image...', verbose)
         sct_labels.remove_missing_labels(Image(ftmp_template_label), Image(ftmp_label)).save(path=ftmp_template_label)
 
-        # Add one label because at least 3 orthogonal labels are required to estimate an affine transformation.
-        add_orthogonal_label(ftmp_label)
-        add_orthogonal_label(ftmp_template_label)
+        # Add a dummy label, because at least 3 orthogonal labels are required to estimate an affine transformation.
+        # -> Pick a dummy label between [1, 127] that doesn't clash with the existing label values.
+        existing_label_vals = {coord.value for coord in img_tmp_label.getCoordinatesAveragedByValue()}
```

Comment:
There's no need to go through the trouble of averaging the coordinates by value, since we immediately throw away all that work, and since the set builder notation will de-duplicate the values just fine:
```suggestion
        existing_label_vals = {coord.value for coord in img_tmp_label.getNonZeroCoordinates()}
```